src/core/components/Paginate/index.js

```diff
@@ -52,28 +61,48 @@ export default class Paginate extends React.Component {
     return makePageNumbers({ start, end });
   }
 
-  makeLink({ currentPage, page, pathname, query, text }) {
-    let child;
+  makeLink({ currentPage, page, pathname, query, text, className }) {
     if (currentPage === page || page < 1 || page > this.pageCount()) {
-      child = text || page;
-    } else {
-      const newQuery = { page, ...query };
-      child = <Link to={{ pathname, query: newQuery }}>{text || page}</Link>;
+      return (
+        <span key={page}
+          className={classNames('Paginator-item', 'disabled', className)}>
+          {text || page}
+        </span>
+      );
     }
-    return <li key={page} className="paginator--item">{child}</li>;
+
+    const newQuery = { page, ...query };
```

Comment:
![it-was-like-that-when-i-got-here](https://cloud.githubusercontent.com/assets/90871/19370137/b25cf0f8-91a1-11e6-9f6b-04e4fd3260d2.jpg)

Indeed! I'll fix it.